corehq/apps/geospatial/const.py

```diff
@@ -4,7 +4,7 @@
 ALGO_AES = 'aes'
 
 # Max number of cases per geohash
-MAX_GEOHASH_DOC_COUNT = 10_000
+MAX_GEOHASH_DOC_COUNT = 10_00
```

Comment:
Maybe make this
```suggestion
MAX_GEOHASH_DOC_COUNT = 1_000
```

Reply:
Addressed in https://github.com/dimagi/commcare-hq/pull/34659/commits/8095702af8a7f672cfc5e39e9bf12fe67a69b42d